.storybook/docs-root.css

```diff
@@ -342,6 +342,12 @@
   overflow: hidden;
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 4 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26403&commitId=dff8fde26992ef963b1372f3c4b9534d4365ff04&env=prod)

✅ There was 2 screenshots added, 2 screenshots removed, 1773 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Text Converged.Typography presets - RTL.normal.chromium.png|0| Added|
|Text Converged.Typography presets.normal.chromium.png|0| Added|
|Text Converged.Typography wrappers - RTL.normal.chromium.png|0| Removed|
|Text Converged.Typography wrappers.normal.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv9"/>